src/System.Net.Security/tests/FunctionalTests/System.Net.Security.Tests.csproj

```diff
@@ -40,6 +40,12 @@
     </Compile>
 
   </ItemGroup>
+  <ItemGroup Condition=" '$(TargetsUnix)' == 'true' "> 
+ 
+    <Compile Include="SslStreamStreamToStreamTest.cs" />   
+    <Compile Include="TestConfiguration.cs" />    
+       
+  </ItemGroup>
```

Comment:
We push the Windows test layout to Linux (this could be changed if we produce the test layout for Linux on Windows), so this will not test what you think it will.